# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/Logging.h
# Commit: 1ac8f10e0025
# Full Hash: 1ac8f10e002518617183160624fa1f40cb66b6e4
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2022-02-12 09:47:43
# Regressor Bug: 1754130
# File Overlap Count: 2
# Description:
#   Bug 1754130 - Implement SourceSurfaceWebgl for faster snapshots of DrawTargetWebgl. r=aosmond
#   
#   Now that each DrawTargetWebgl shares the same WebGL context, we can efficiently draw snapshots
#   of one DrawTargetWebgl to another without requiring any readback or error-prone driver-provided
#   shared context/resource mechanism. We just need to simply pass the WebGL texture from one target
# ==============================================================================

diff -r 3d56b847ddf9 -r 1ac8f10e0025 gfx/2d/Logging.h
--- a/gfx/2d/Logging.h	Fri Feb 11 15:36:29 2022 +0000
+++ b/gfx/2d/Logging.h	Fri Feb 11 15:36:30 2022 +0000
@@ -694,6 +694,9 @@
         case SurfaceType::DATA_MAPPED:
           mMessage << "SurfaceType::DATA_MAPPED";
           break;
+        case SurfaceType::WEBGL:
+          mMessage << "SurfaceType::WEBGL";
+          break;
         default:
           mMessage << "Invalid SurfaceType (" << (int)aType << ")";
           break;